libs/coin-evm/src/api/node/ledger.ts

```diff
@@ -223,7 +231,6 @@ export const getFeeData: NodeApi["getFeeData"] = async (currency, transaction) =
       ...currency,
       ethereumLikeInfo: {
         ...currency.ethereumLikeInfo!,
-        gasTracker: { type: "ledger", explorerId: node.explorerId },
```

Comment:
Why removing it?

Reply:
We get the gasTracker information directly in `getGasOptions`

Reply:
No actually @sprohaszka-ledger has a point, this removal is kinda problematic. 
This overriding mechanism is here to help with the separation of concerns of the 3 "backends" we can rely on with an EVM: A node, and explorer and a gas tracker.

Here, the goal was to say that Ledger "as a node" should be able to provide the exact same methods as a classic RPC node. But Ledger's backend isn't providing all features for now. In order to fake the behaviour of an eth_gasPrice/eth_feeHistory RPC request, we go get the result of the Ledger `gasTracker` (no matter what the configuration of the currency is) and return it as if it was an eth_gasPrice/eth_feeHistory RPC response.

Here if the configuration of the coin is set to use let's say Blocknative gas tracker (not implemented today, I know), using Ledger "as a node" would result in using Blocknative when requesting fees, and not just Ledger anymore.

So we need a way to enforce this ledger config when using `getGasOptions`.